README.md

```diff
@@ -369,22 +371,24 @@ Compose.prototype.map = function(f) {
 #### `traverse` method
 
 ```hs
-traverse :: Applicative f, Traversable t => t a ~> (a -> f b, c -> f c) -> f (t b)
+traverse :: Applicative f, Traversable t => t a ~> (TypeRep f, a -> f b) -> f (t b)
 ```
 
 A value which has a Traversable must provide a `traverse` method. The `traverse`
 method takes two arguments:
 
-    u.traverse(f, of)
+    u.traverse(A, f)
```

Comment:
I didn't like the old order, but I couldn't justify a breaking change to scratch my itch. But if we're making a breaking change anyway we can flip the parameters while we're at it. :)

Out of interest, why do you prefer the new order?